framework/src/play/db/jpa/Blob.java

```diff
@@ -3,41 +3,31 @@
 import java.io.File;
 import java.io.FileInputStream;
 import java.io.InputStream;
-import java.io.Serializable;
-import java.sql.PreparedStatement;
-import java.sql.ResultSet;
-import java.sql.SQLException;
-import java.sql.Types;
-import java.util.Objects;
-
-import org.hibernate.HibernateException;
-import org.hibernate.engine.spi.SharedSessionContractImplementor;
-import org.hibernate.type.StringType;
-import org.hibernate.usertype.UserType;
 
 import play.Play;
 import play.db.Model.BinaryField;
 import play.exceptions.UnexpectedException;
 import play.libs.Codec;
 import play.libs.IO;
 
-public class Blob implements BinaryField, UserType {
```

Comment:
Why did you remove the UserType implementation and define the new class `BlodType` ?